.gitmodules

```diff
@@ -7,4 +7,4 @@
 [submodule "lib/memview-sol"]
 	path = lib/memview-sol
 	url = https://github.com/summa-tx/memview-sol
-	branch = main
+	branch = v2.1.1
```

Comment:
What's the reason to change this? I had issues with setting the branch to `v2.1.1` last week and could only get it working with `main`

Reply:
Oh really? Okay yah this is the stupidest thing. Sometimes it works for me on main, sometimes I need to specify the version. I had explained it to the foundry team at the beginning of January and they said it was a bug and were going to fix it. For now I'll remove this change, but hopefully we can get on the same page with our lib/ soon lol